venter/serializers.py

```diff
@@ -33,14 +33,15 @@ class Meta:
         model = Complaints
         fields = (
             'id', 'created_by', 'description', 'report_date', 'status', 'latitude',
-            'longitude', 'location_description', 'tags', 'comments', 'users_up_voted', 'images'
+            'longitude', 'location_description', 'tags', 'comments', 'authorities', 'users_up_voted', 'images'
```

Comment:
Why do we want authorities in the serializer again?

Reply:
Removing it from there causes issues in tests written previously.

Reply:
Which tests? That doesn't really make any sense since the authorities model didn't exist before this to test it in the first place.

Reply:
.

Reply:
Reverted, tests didn't show any errors